TMS.Net07.Homework6.ShapeDraw/TMS.Net07.Homework6.ShapeDraw/ConsoleRusDescriptionDrawer.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework6.ShapeDraw
+{
+    public class ConsoleRusDescriptionDrawer : Drawer
+    {
+        public override void Draw(Shape shape)
+        {
+            Console.WriteLine($"{shape.GetType()}");
+            var points = shape.GetPoints();
+            
+            if (shape is Circle circle)  
+            {
+                circle.A.X = points[0];
```

Comment:
We don't need to do it at all: we are getting points from the shape and after that reassign them back - we don't change anything.
After making `(shape is Circle circle)` we don't create a new object in memory, we just create a one more pointer to it. We just check if shape is a circle and if it is, we create a new pointer of `Circle` type and continue working with `shape` as with `Circle`.